src/main.cpp

```diff
@@ -675,36 +676,33 @@ void GetGlobalStatus()
 
 std::string GetCurrentOverviewTabPoll()
 {
-    std::string poll = "";
-    std::string sMessageKey = ExtractXML(msPoll, "<MK>", "</MK>");
-    std::string sPollExpiration = ExtractXML(msPoll, "<EXPIRATION>", "</EXPIRATION>");
-    uint64_t uPollExpiration = 0;
-    // Alerts are displayed as polls but do not have an expiration
-    if(sPollExpiration.empty())
-    {
-        uPollExpiration = pindexBest->nTime;
+    AssertLockHeld(MinerStatus.lock);
+
+    // The global msPoll variable contains the poll most-recently published to
+    // the network. If it hasn't expired, return the title of this poll:
+    if (PollIsActive(msPoll)) {
+        return ExtractXML(msPoll, "<MK>", "</MK>").substr(0, 80);
     }
```

Comment:
Note: need to investigate how the alerts plug into the polls--I couldn't see how this works.

Reply:
Looks like that logic was introduced in #1250.